backend/database.js

```diff
@@ -35,12 +36,19 @@ export default (sbp('sbp/selectors/register', {
         try {
           const entry = await sbp('chelonia/db/getEntry', currentHEAD)
           const json = `"${strToB64(entry.serialize())}"`
-          if (currentHEAD !== hash) {
-            this.push(prefix + json)
-            currentHEAD = entry.message().previousHEAD
-            prefix = ','
-          } else {
-            this.push(prefix + json + ']')
+
+          if (currentHEAD === hash) {
+            isMet = true
+          } else if (isMet) {
+            offset--
+          }
+
+          this.push(prefix + json)
+          currentHEAD = entry.message().previousHEAD
+          prefix = ','
+
+          if (isMet && offset <= 0) {
```

Comment:
This is wrong... and this is why I saw these errors:

```
read(): Error during getEntry: no entry for null!: ChelErrorDBConnection: Error during getEntry: no entry for null!
    at Object.chelonia/db/getEntry (group-income/shared/domains/chelonia/db.js:53:13)
```

But not only that, this function shouldn't have been modified. You should be using `'backend/db/streamEntriesBefore'` instead of loading all of the events that we already have (+ 20) each time the user scrolls.

Reply:
I should remove this offset parameter here in this `streamEntriesSince` API and should make a new API `streamEntriesBetween`.